pkg/kv/kvserver/allocator_scorer.go

```diff
@@ -46,16 +48,11 @@ const (
 	// https://www.eecs.harvard.edu/~michaelm/postscripts/mythesis.pdf.
 	allocatorRandomCount = 2
 
-	// maxFractionUsedThreshold: if the fraction used of a store descriptor
-	// capacity is greater than this value, it will never be used as a rebalance
-	// or allocate target and we will actively try to move replicas off of it.
-	maxFractionUsedThreshold = 0.95
-
 	// rebalanceToMaxFractionUsedThreshold: if the fraction used of a store
```

Comment:
This should also go along to base.go, if you have the other disk fullness check there.

Reply:
See comment above.